docs/refactoring-cli-node-module.md

```
@@ -0,0 +1,525 @@
+**Files Affected:**
+- `src/nerve/frontends/cli/server/node.py`
+- `src/nerve/frontends/cli/server/session.py`
+- `src/nerve/frontends/cli/server/__init__.py`
+- `src/nerve/frontends/cli/server/graph.py`
+
+#### Current State
+
+Two different error handling patterns are used inconsistently:
+
+**Pattern 1: `error_exit()` from output.py** (used in node.py)
+```python
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)